tools/doc/type-parser.js

```diff
@@ -38,8 +43,9 @@ module.exports = {
       typeText = typeText.trim();
       if (typeText) {
         let typeUrl = null;
-        if (jsPrimitives.indexOf(typeText) !== -1) {
-          typeUrl = jsPrimitiveUrl + '#' + typeText + '_type';
+        const primitive = jsPrimitives[typeText];
+        if (primitive !== undefined) {
```

Comment:
Checking with `undefined` might be more precise and meaningful here, because here we are checking if this primitive type by the `typeText` is undefined, this is actually my original thoughts.

Both looks good to me, so @lpinca if you have an objection about undefined, I will change to follow you :)